docs/src/content/v1-migration/intersection.md

```diff
@@ -0,0 +1,75 @@
+# Typing
+
+The typing for the `other` operand was softened so that it could accept any
```

Comment:
**nit:** softened -> widened (which is the term ts uses in the docs)

```suggestion
The typing for the `other` operand was widened so that it could accept any
```